Cargo.lock

```diff
@@ -0,0 +1,2626 @@
+# This file is automatically @generated by Cargo.
```

Comment:
How come we have to check in this file?  Isn't it included in the deltachat-core-rust repo already?  Or is this a different file?

Reply:
good catch.